charts/spire/README.md

```diff
@@ -78,13 +99,86 @@ Now you can interact with the Spire agent socket from your own application. The
 | Repository | Name | Version |
 |------------|------|---------|
 | file://./charts/spiffe-csi-driver | spiffe-csi-driver | 0.1.0 |
-| file://./charts/spiffe-csi-driver | upstream-spiffe-csi-driver(spiffe-csi-driver) | 0.1.0 |
 | file://./charts/spiffe-oidc-discovery-provider | spiffe-oidc-discovery-provider | 0.1.0 |
 | file://./charts/spire-agent | spire-agent | 0.1.0 |
-| file://./charts/spire-agent | upstream-spire-agent(spire-agent) | 0.1.0 |
```

Comment:
Revert, this was added by another PR adding the upstream spire things.

Reply:
Have reverted, these lines are now in the README file.